src-java/rule-manager/rule-manager-api/src/main/java/org/openkilda/match/MatchType.java

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2021 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.match;
+
+public enum MatchType {
```

Comment:
Didn't get why for we need this class. We already have `Field` (for IN_PORT and METADATA). You can add `isMasked` into `Match` to coverr (FIELD_MASKED/FIELD_EXACT).

Reply:
Reworked.